example/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I know that's an example, but assuming people will copy those file to create their projects: replace all the content with `"@vtex/prettier-config"` 🙏